doc/development_guide/index.rst

```diff
@@ -27,6 +27,17 @@ The FSF GNAT and all Ada dependencies can be installed using Alire.
    $ source venv/bin/activate
    $ make install_devel
 
+The repository contains configuration files for Visual Studio Code which enable most linters available from the Makefile and coverage reports integrated into the IDE.
+To make use of the coverage data, the `Coverage Gutter <https://github.com/ryanluker/vscode-coverage-gutters>` extension needs to be installed.
```

Comment:
Coverage Gutters just tells me "No Coverage", even after executing the "Python Coverage" task. Is there some configuration needed so that this extension finds the created `build/cov.xml`?

Reply:
Just checked - I have no configuration beyond what's check in to the workspace. Note, that it will always say "No Coverage" unless you select a Python file that a coverage analysis was done for. Unfortunately there is no global view outside the textual logs. As I said - that solution is not perfect, yet.

Reply:
I see. It works for me now. I guess I have actually looked at a file without coverage information.